rule_test.go

```diff
@@ -325,15 +325,15 @@ func runRuleListFiltered(t *testing.T, family int, srcNet, dstNet *net.IPNet) {
 		},
 		{
 			name:       "returns rules filtered by Mask",
-			ruleFilter: &Rule{Mask: 0x5},
+			ruleFilter: &Rule{Mask: &[]uint32{0x5}[0]},
```

Comment:
this is not elegant or idiomatic in my opinion, please write a helper function:
```go
func maskAsPtr(mask uint32) *uint32 {
        return &mask
}
```

Then this becomes `Mask: maskAsPtr(0x5)`

Reply:
`&[]uint32{0x5}[0]` is the only way I know to get a pointer to a simple type in a single line of code. 
It can simplify a lot of code in unit tests, especially if there isn't an existing function to get a pointer to the corresponding simple type.

The PR has been merged, so I'm thinking of not making any more changes for now.